tests/telemetry/test_telemetry.py

```diff
@@ -19,21 +25,30 @@ def get_list_stdout(cmd_out):
         out_list.append(result)
     return out_list
 
+def setup_telemetry_forpyclient(duthost):
+    """ Set client_auth=false. This is needed for pyclient to sucessfully set up channel with gnmi server.
+        Restart telemetry process
+    """
+    client_auth_out = duthost.shell('/usr/bin/redis-cli -n 4 hget "TELEMETRY|gnmi" "client_auth"', module_ignore_errors=False)['stdout_lines']
```

Comment:
change this to db client?

Reply:
replaced all redis-cli with sonic-db-cli